events/alcohol-advice.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Event Alcohol Advice | Ruby New Zealand Incorporated</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,900" rel="stylesheet">
```

Comment:
Could we please use the same font as the main page? It's [Poppins](https://fonts.google.com/specimen/Poppins)